hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -264,11 +265,10 @@ protected HStore(final HRegion region, final ColumnFamilyDescriptor family,
     region.getRegionFileSystem().createStoreDir(family.getNameAsString());
 
     // set block storage policy for store directory
-    String policyName = family.getStoragePolicy();
-    if (null == policyName) {
-      policyName = this.conf.get(BLOCK_STORAGE_POLICY_KEY, DEFAULT_BLOCK_STORAGE_POLICY);
-    }
-    region.getRegionFileSystem().setStoragePolicy(family.getNameAsString(), policyName.trim());
+    this.policyName = Optional.ofNullable(family.getStoragePolicy())
```

Comment:
is there any change in logic here? Seems same as before just that we are using lambdas now?

Reply:
yes, no logic change. just to simplify assigning values ​​to "this.policyName"